proto/streamlit/proto/Element.proto

```diff
@@ -55,14 +56,15 @@ message Element {
   // An element can be one of the following element types.
   oneof type {
     Alert alert = 30;
+    ArrowTable table = 35;
     Audio audio = 13;
     Balloons balloons = 12;
     BokehChart bokeh_chart = 17;
     Button button = 19;
     Chart chart = 4;
     Checkbox checkbox = 20;
     DataFrame data_frame = 3;
-    DataFrame table = 11;
+    DataFrame old_table = 11;
```

Comment:
If we make this obsolete later, we would have to leave it in here, correct?

Reply:
The new methods will use the same name (`table`), so no. They will just be overwritten.